frontend/src/containers/ApplyPage/ApplyInfo/ApplyInfo.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+
+import { Link } from '../../../components'
+
+class ApplyInfo extends Component {
+  render() {
+    return (
+      <div>
+        <p>We're accepting applications all the time. It's great if your responses to these questions are over 3 sentences long, but make sure you don't write an essay!</p>
+
+        <br />
+
+        <p>Check out our <Link href="/example_applications">example applications</Link> if you want some help!</p>
```

Comment:
@paked just chatted in person about this. This page will be added when we migrate all the static pages from https://github.com/hackclub/website